### PR TITLE
fix: Remove headers

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
Remove header since that is  a dict with elements that are not serializable

### CODE REVIEW

**Review by mariajgrimaldi — APPROVED** [review]

File: eox_tagging/api/v1/viewset.py [hunk 1]
```diff
@@ -204,10 +204,10 @@ def create(self, request, *args, **kwargs):
         """
 
         @audit_method(action="eox_tagging-api-v1-viewset:tagviewset-create")
-        def audited_create(headers, body):  # pylint: disable=unused-argument
+        def audited_create(body):  # pylint: disable=unused-argument
             return super(TagViewSet, self).create(request, *args, **kwargs)
 
-        return audited_create(headers=request.headers, body=request.data)
```
Comment (MoisesGSalas) [inline eox_tagging/api/v1/viewset.py:210]: What are the elements that are not serializable? I thought all the elements were strings. I think the problem is headers is not a real dict, so maybe something like `dict(request.headers)` could work.

Comment (andrey-canon) [reply eox_tagging/api/v1/viewset.py:210]: I will test that

Comment (andrey-canon) [reply eox_tagging/api/v1/viewset.py:210]: The problem is not the headers, the headers variable is a dict the problem is one element inside the dict which is not a dict, with your suggestion is the same

Comment (MoisesGSalas) [reply eox_tagging/api/v1/viewset.py:210]: In that case go ahead

**Review by MoisesGSalas — APPROVED** [review]

**Pull request merged**